theano/tensor/nnet/abstract_conv.py

```diff
@@ -1690,9 +1690,9 @@ def perform(self, node, inp, out_):
             img = new_img
         if not self.filter_flip:
             if self.unshared is True:
```

Comment:
I've now added a check in the `init` function of `BaseAbstractConv`. Will that do?